<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="login_frame">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>300</width>
      <height>400</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="login_label">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>110</y>
       <width>250</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="username_input">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>200</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="login_button">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>230</y>
       <width>200</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="login_error_label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>260</y>
       <width>200</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Error</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="header_frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>50</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="header_name">
     <property name="geometry">
      <rect>
       <x>476</x>
       <y>15</y>
       <width>311</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="crash_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>15</y>
       <width>60</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>CRASH</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="crash_frame">
    <property name="geometry">
     <rect>
      <x>-15</x>
      <y>75</y>
      <width>800</width>
      <height>550</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="crash_amount_frame">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>470</width>
       <height>260</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="crash_amount">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>450</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QFrame" name="crash_history_frame">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>471</width>
        <height>50</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="crash_history_1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>5</y>
         <width>80</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>History 1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="crash_history_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>5</y>
         <width>80</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>History 2</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="crash_history_3">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>5</y>
         <width>80</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>History 3</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="crash_history_4">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>5</y>
         <width>80</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>History 4</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="crash_history_5">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>5</y>
         <width>80</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>History 5</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QPushButton" name="crash_bet_button">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>420</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Bet</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="crash_bet_amount">
     <property name="geometry">
      <rect>
       <x>234</x>
       <y>345</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="crash_bet_money_sign">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>345</y>
       <width>25</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>$</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="crash_bet_label">
     <property name="geometry">
      <rect>
       <x>234</x>
       <y>345</y>
       <width>130</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="crash_bet_message">
     <property name="geometry">
      <rect>
       <x>165</x>
       <y>470</y>
       <width>250</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Error</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QScrollArea" name="crash_players_scroll">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>40</y>
       <width>270</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(60, 60, 60);
border: 0px solid white;</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>270</width>
        <height>431</height>
       </rect>
      </property>
      <widget class="QLabel" name="crash_players_label">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>65</y>
         <width>240</width>
         <height>5000</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
       </property>
      </widget>
      <widget class="QLabel" name="crash_players_title">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>10</y>
         <width>130</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Players</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </widget>
    <zorder>crash_amount_frame</zorder>
    <zorder>crash_bet_button</zorder>
    <zorder>crash_bet_money_sign</zorder>
    <zorder>crash_bet_label</zorder>
    <zorder>crash_bet_amount</zorder>
    <zorder>crash_bet_message</zorder>
    <zorder>crash_players_scroll</zorder>
   </widget>
   <zorder>header_frame</zorder>
   <zorder>login_frame</zorder>
   <zorder>crash_frame</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
